en/08_Changelogs/5.3.0.md

```diff
@@ -51,6 +52,26 @@ Out of the box `silverstripe/assets` provides the new [`InterventionImageFileCon
 
 See [convert a file to a different format](/developer_guides/files/file_manipulation/#file-conversion) for more details.
 
+### Improve customisability of rendered images {#image-rendering}
+
+We've aligned how images added to an HTMLEditorField and SS template images are rendered. The markup up for both `<img>` tag is now generated from the same template file. This will make it easier to customise how images are rendered on your site.
+
+Simply add a `templates/SilverStripe/Assets/Storage/DBFile_Image.ss` file to your Silverstripe CMS theme.
```

Comment:
```suggestion
We've aligned how images added to an [`HTMLEditorField`](api:SilverStripe\Forms\HTMLEditor\HTMLEditorField) and images inside SS templates are rendered. The markup up for both `<img>` tags are now generated from the same template file. This makes it easier to customise how images by creating a `templates/SilverStripe/Assets/Storage/DBFile_Image.ss` file to your Silverstripe CMS theme.
```

Reply:
I did the first paragraph fix. Not the second.